tozny/resource_realm_group.go

```diff
@@ -76,10 +77,132 @@ func resourceRealmGroup() *schema.Resource {
 				Computed:    true,
 				ForceNew:    true,
 			},
+			"access_policies": {
```

Comment:
to be future proof semantically I think `access_policy`is called for here, because when writing terraform if the api/provider does support defining multiple access policies this makes sense

```
access_policy {}
access_policy {}
```

vs this which looks odd

```
access_policies {}
access_policies {}
```